securing_apps/topics/oidc/java/java-adapter-config.adoc

```diff
@@ -197,6 +197,10 @@ client-key-password::
 always-refresh-token::
   If _true_, the adapter will refresh token in every request.
```

Comment:
I suggest just adding to the end of this line something like:
`If _true_, the adapter will refresh token in every request (warning - enabling this will result in a request to ${project_name} for every request to your application).`